### PR TITLE
add default coreos version to config.yaml

### PR DESCRIPTION
**Pull request by teemow**
the default coreos version has been hardcoded before. this version is used to bootstrap machines (first boot) and needs to be defined.

### CODE REVIEW

**Comment by teemow** [comment]
ping @hectorj2f @JosephSalisbury

File: pxemgr/ipxe_handlers.go [hunk 1]
```diff
@@ -617,19 +615,23 @@ func (mgr *pxeManagerT) welcomeHandler(w http.ResponseWriter, r *http.Request) {
...
-	coreOSversion := defaultCoreOSVersion
+	coreOSversion := mgr.config.DefaultCoreOSVersion
+	fmt.Println("default: ", coreOSversion)
 
 	host, exists := mgr.hostFromSerialVar(r)
 	if exists {
+		fmt.Println("host from serial: ", host.CoreOSVersion)
+
+		if version, exist := host.Overrides["CoreOSVersion"]; exist {
+			fmt.Println("send host override: ", version.(string))
```
Comment (hectorj2f) [inline pxemgr/ipxe_handlers.go]: @teemow the code looks good to me but maybe we want messages which are more descriptive. "default: ". "send default"... maybe something like "coreos version:" ...

**Comment by hectorj2f** [comment]
LGTM, minor comment

File: pxemgr/ipxe_handlers.go (hunk 1 above)
Comment (teemow) [inline pxemgr/ipxe_handlers.go]: hehe, sry these are anyway just debug messages. but yes, I'll remove them and add proper logging.

Comment (teemow) [inline pxemgr/ipxe_handlers.go]: done

**Pull request merged**